presto-main/src/main/java/io/prestosql/metadata/Metadata.java

```diff
@@ -53,20 +53,14 @@
 
 public interface Metadata
 {
-    void verifyComparableOrderableContract();
```

Comment:
This might cause a lot rebasing issues to existing PRs and PRs which are not yet created (are not yet open sourced).

I like the change, but I am a bit afraid...